pkg/network/tracer/tracer.go

```diff
@@ -540,6 +545,7 @@ func (t *Tracer) GetStats() (map[string]interface{}, error) {
 
 	stateStats := t.state.GetStats()
 	conntrackStats := t.conntracker.GetStats()
+	atomic.StoreInt64(&t.conntrackerSamplingPct, conntrackStats["sampling_pct"])
```

Comment:
Since this stat is driving behavior now, the name should probably be a constant to prevent mismatch.

Reply:
Alternatively, something like `IsSampling()` could be part of the interface. It feels a bit brittle to rely upon statistics from a separate component.

Reply:
OK, I can add a `IsSampling` method